target/cheri-common/cheri-lazy-capregs.h

```diff
@@ -317,6 +317,27 @@ static inline target_ulong get_capreg_tag(CPUArchState *env, unsigned regnum)
     tcg_abort();
 }
 
+static inline target_ulong get_capreg_tag_revokable(CPUArchState *env, unsigned regnum) {
```

Comment:
```suggestion
static inline bool get_capreg_tag_revokable(CPUArchState *env, unsigned regnum) {
```

Reply:
Should this not have the same type as get_capreg_tag?

Reply:
I thought get_capreg_tag was bool... I guess in that case it's fine.

Reply:
Will keep the type the same then.